src/containers/HomePage/IntroBlock.js

```diff
@@ -23,15 +23,15 @@ class IntroBlock extends React.Component {
         </div>
         <div className="fl-m fl-l w-33-m w-33-l">
           <div className="pa4 pa4-m">
-            <h3 className="f3">We are seeking photos</h3>
+    <h3 className="f3">2.) We are seeking photos</h3>
```

Comment:
Maybe I didn't explain things right but we didn't want to change this line.... please change it back to what it was, thanks. Step 2 will come later in another modification.